content/_data/changelogs/weekly.yml

```diff
@@ -982,6 +982,29 @@
       pull: 2956
     # pull: 2971 not notable enough
     # pull: 2968 not notable enough
+- version: "2.75"
+  date: 2017-08-20
+  changes:
+    - type: rfe
+      message: >
+        Update Windows service wrapper from 2.1.0 to 2.1.2 and Windows Agent Installer from 1.9 to 1.9.1.
```

Comment:
I know we go back and forth on this kind of entry, but if we are happy to just link to Groovy changelogs for https://jenkins.io/changelog/#v2.61 I don't see why this would need to be different. The components have externally maintained changelogs, so we link to them. Anyone caring enough to look will be able to read what changed, and it allows you as maintainer of the component to independently revise the changelog as needed without duplicating work.